docs/workshops/interviews/technical/workshop.md

```diff
@@ -0,0 +1,262 @@
+---
+id: workshop
+title: Technical Interviews
+sidebar_label: Technical Interviews
+---
+
+# Technical Interviews
```

Comment:
Sorry to be this person but - could we get some Learning Objectives here? What exactly are we trying to achieve with this workshop?